### PR TITLE
✨ redesigned gallery item as default

### LINKED ISSUE #4696: Make new Gallery Item detail as default
It's time to remove redesign switch and make it default to catch last 🐛🪲

### PR DESCRIPTION
**Pull request by petersopko**
## PR Type

- [ ] Bugfix
- [x] Feature
- [ ] Refactoring

## Context

- [x] Closes #4696 
- [ ] Requires deployment <>

#### Before submitting pull request, please make sure:

- [x] My contribution builds **clean without any errors or warnings**
- [x] I've merged recent default branch -- **main** and I've no conflicts
- [x] I've tried to respect high code quality standards
- [x] I've didn't break any original functionality
- [x] I've posted a screenshot of demonstrated change in this PR

#### Optional

- [x] I've tested it at </bsx/collection>
- [x] I've tested PR on mobile
- [x] I've written unit tests 🧪
- [x] I've found edge cases

#### Had issue bounty label?

- [x] Fill up your KSM address: [Payout](https://beta.kodadot.xyz/transfer/?target=FNyt7T1xdbhN7dagf7yGYCRJuE3R45VmTCE3tjzy1rKxa7y)

#### Community participation

- [x] [Are you at KodaDot Discord?](https://discord.gg/35hzy2dXXh)

## Screenshot 📸

- [x] My fix has changed **something** on UI; a screenshot is best to understand changes for others.
image

### CODE REVIEW

**Comment by yangwao** [comment]
so many changes 🤔

**Comment by petersopko** [comment]
> so many changes 🤔

not good ? I thought I'd enable it on movr/glmr as well.

some minor changes related to how offers are (not) displayed there. 

lmk if something needs to be removed / added.

File: components/gallery/GalleryItem.vue [hunk 1]
```diff
@@ -117,6 +122,10 @@ const route = useRoute()
 const router = useRouter()
 const message = ref('')
 
+const offersDisabled = computed(() => {
```
[the remark this answers is not in the document]
Comment (petersopko) [reply components/gallery/GalleryItem.vue]: looks better, good idea 👍🏻

**Review by roiLeo — COMMENTED** [review]
- can we have Vue3 style implementation on `pages/...`?
- is it normal that you remove image generation for seo?

**Comment by yangwao** [comment]
@petersopko 👀

**Comment by petersopko** [comment]
added dynamic routing, I've spent some time trying to go the "nuxt 3 way", however I couldn't get the square brackets to work so for now it's `_prefix/gallery/_id.vue`

**Comment by petersopko** [comment]
> > // SEO settings is back
> 
> there is already some SEO attributes inside GalleryItem.vue https://github.com/kodadot/nft-gallery/blob/main/components/gallery/GalleryItem.vue#L134-L156 could you check that, maybe we dont need this section https://github.com/petersopko/nft-gallery/blob/fix/4696-gallery-item-redesign-launch/pages/_prefix/gallery/_id.vue#L10

I've tested the SEO inside of GalleryItem; it is working (i.e., the image is being responsive and with correct aspect ratio). Still many SEO errors, but I don't think it's related to this PR or issue. Maybe follow up to check SEO score and adjust?

**Comment by yangwao** [comment]
pay 30 usd

**Comment by yangwao** [comment]
😍 Perfect, I’ve sent the payout 
💵 $30 @ 34.24 USD/KSM ~ 0.876 $KSM
🧗 [FNyt7T1xdbhN7dagf7yGYCRJuE3R45VmTCE3tjzy1rKxa7y](https://kusama.subscan.io/account/FNyt7T1xdbhN7dagf7yGYCRJuE3R45VmTCE3tjzy1rKxa7y)
🔗 [0x6a73a54f930d88480081d8a9df405a31099689610851dec07b39a9dbdf73bb5f](https://kusama.subscan.io/extrinsic/0x6a73a54f930d88480081d8a9df405a31099689610851dec07b39a9dbdf73bb5f)

🪅 Let’s grab another issue and get rewarded! 
🪄 [github.com/kodadot/nft-gallery/issues](https://github.com/kodadot/nft-gallery/issues/)

**Comment by petersopko** [comment]
> 😍 Perfect, I’ve sent the payout 💵 $30 @ 34.24 USD/KSM ~ 0.876 $KSM 🧗 [FNyt7T1xdbhN7dagf7yGYCRJuE3R45VmTCE3tjzy1rKxa7y](https://kusama.subscan.io/account/FNyt7T1xdbhN7dagf7yGYCRJuE3R45VmTCE3tjzy1rKxa7y) 🔗 [0x6a73a54f930d88480081d8a9df405a31099689610851dec07b39a9dbdf73bb5f](https://kusama.subscan.io/extrinsic/0x6a73a54f930d88480081d8a9df405a31099689610851dec07b39a9dbdf73bb5f)
> 
> 🪅 Let’s grab another issue and get rewarded! 🪄 [github.com/kodadot/nft-gallery/issues](https://github.com/kodadot/nft-gallery/issues/)

didn't work 👀

**Comment by yangwao** [comment]
pay 30

**Comment by yangwao** [comment]
😍 Perfect, I’ve sent the payout 
💵 $30 @ 35.11 USD/KSM ~ 0.854 $KSM
🧗 [FNyt7T1xdbhN7dagf7yGYCRJuE3R45VmTCE3tjzy1rKxa7y](https://kusama.subscan.io/account/FNyt7T1xdbhN7dagf7yGYCRJuE3R45VmTCE3tjzy1rKxa7y)
🔗 [0x4255b6c44416267d48b9d2ed0bf50c8d168f1027a98b3833808237b493700d52](https://kusama.subscan.io/extrinsic/0x4255b6c44416267d48b9d2ed0bf50c8d168f1027a98b3833808237b493700d52)

🪅 Let’s grab another issue and get rewarded! 
🪄 [github.com/kodadot/nft-gallery/issues](https://github.com/kodadot/nft-gallery/issues/)

**Pull request merged**